lxdock/guests/base.py

```diff
@@ -175,6 +182,10 @@ def copy_directory(self, host_path, guest_path):
         self.run(['tar', '-xf', guest_tar_path, '-C', str(guest_path)])
         self.run(['rm', '-f', str(guest_tar_path)])
 
+    def is_provisioned(self):
+        """ Returns a boolean indicating if the container is provisioned. """
+        return self.lxd_container.config.get('user.lxdock.provisioned') == 'true'
+
```

Comment:
@hsoft I think this is mixing up `Guest` and `Container` classes? I see the usage of `Guest.is_provisioned()` method in Provisioner classes, but conceptually this only concerns LXD and is independent from containers' operating systems. 

How about: not passing the provisioned containers' guests into the Provisioner class, inside `Container.provision()` and `Project.provision()`?